apps/champions/lib/champions/config.ex

```diff
@@ -133,4 +132,38 @@ defmodule Champions.Config do
       )
     end)
   end
+
+  @seconds_in_day 86_400
```

Comment:
I think this should be at the beginning of the file

Reply:
It's only used for the function below it, why would you move it to the top of the file 🤔 ? You gain nothing + you have to scroll all the way back up to check the value